### PR TITLE
Wait for file completed in fuse getAttr (#8192)

### PR DESCRIPTION
**Pull request by calvinjia**
cherry-pick of Alluxio/alluxio#8192
cherry-pick commits are:
change fuse get attr : Alluxio/alluxio@fb36eea63268
change the error message : Alluxio/alluxio@9c78eaa12b36
update status after file completed : Alluxio/alluxio@70eb188fa485
fix the getAttr error : Alluxio/alluxio@c270a101fad6

### CODE REVIEW

**Comment by gpang** [comment]
jenkins, test this please

**Comment by gpang** [comment]
```
Failed tests: 
  AlluxioFuseFileSystemTest.getattr:153 expected:<0> but was:<-14>
```

**Comment by calvinjia** [comment]
@gpang looks like I need this change first #9499 (though this one might also need this current one?)

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java [hunk 1]
```diff
@@ -280,8 +280,13 @@ public int getattr(String path, FileStat stat) {
       if (!mFileSystem.exists(turi)) {
         return -ErrorCodes.ENOENT();
       }
-      final URIStatus status = mFileSystem.getStatus(turi);
-
+      URIStatus status = mFileSystem.getStatus(turi);
+      if (!status.isCompleted()) {
+        if (!waitForFileCompleted(turi)) {
```
Comment (liuhongtong) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java]: Should we consider the mOpenFiles?

Comment (calvinjia) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java]: Do you mean to check if it is in `mOpenFiles`? The file could be closed during the wait period.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**